trello_webhooks/models.py

```diff
@@ -319,6 +321,19 @@ def template(self):
         """Return full path to render template, based on event_type."""
         return 'trello_webhooks/%s.html' % self.event_type
 
+    @property
```

Comment:
Beware of marking something that does a network call (esp. one that could take a long time - see comment below) as a property. Method preferred in this case.